release-notes/9.0/known-issues.md

```diff
@@ -39,3 +40,15 @@ private void ReactiveWindow_ContentRendered(object sender, System.EventArgs e)
 ```
 
 The behavior will be **fixed in .NET 9 RC1** release.
+
+## .NET SDK
+
+### .NET 9.0.100-RC.1 will fail when used with 17.11
+
+There is an incompatible dependency between a file loaded from the SDK and the dependencies that Visual Studio includes.
+
+`
+NETSDK1060 Error reading assets file: Error loading lock file '...\obj\project.assets.json' : Could not load file or assembly 'System.Text.Json, Version=8.0.0.4, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.
+`
+
+This behavior will be **fixed in an upcoming Visual Studio 17.11.x** release
```

Comment:
17.11.3 was released yesterday, does it include the fix?

Reply:
Is there any known workarounds? Even if too hacky like going into MSBuild installation or something and adding a binding redirect somewhere, etc?

Reply:
We're working on some additional guidance.

Reply:
Fixed on 17.11.4